hedera-node/hedera-app/src/xtest/java/contract/UpdatesExpiryXTest.java

```diff
@@ -79,6 +106,16 @@ protected void doScenarioOperations() {
                         .array()),
                 assertSuccess());
 
+        // Should throw `INVALID_SIGNATURE`
+        runHtsCallAndExpectOnSuccess(
+                SENDER_BESU_ADDRESS,
+                Bytes.wrap(UpdateExpiryTranslator.UPDATE_TOKEN_EXPIRY_INFO_V2
+                        .encodeCallWithArgs(
+                                A_TOKEN_ADDRESS, Tuple.of(EXPIRY_TIMESTAMP, SENDER_HEADLONG_ADDRESS, AUTO_RENEW_PERIOD))
+                        .array()),
+                output -> assertEquals(
+                        Bytes.wrap(ReturnTypes.encodedRc(INVALID_SIGNATURE).array()), output, "Wrong key"));
+
```

Comment:
Consider doing the tests for both V1 and V2, as it's currently unclear why we use V1 for some cases and V2 for others. Same suggestion for `UpdatesXTest.java` also.
For this class perhaps you could extract the operations into another method that takes the Function (either V1 or V2) as a parameter and call it for both scenarios in `doScenarioOperations`, this could save some boilerplate.

Reply:
The tests are already present for all the signature versions, if I understood your suggestion correctly..

Reply:
Sorry for any confusion earlier. My comment was more about the approach in `UpdatesExpiryXTest` & `UpdatesXTest` rather than this specific test case. Currently, it's unclear why we choose using V1 to check for INVALID_EXPIRATION_TIME, INVALID_RENEWAL_PERIOD, and INVALID_TOKEN_ID. We could consider incorporating tests for both, since that could enhance clarity.